app/models/server/models/Sessions.js

```diff
@@ -1,5 +1,374 @@
 import { Base } from './_Base';
 
+export const aggregates = {
```

Comment:
any reason to export a whole object instead of individual functions?

Reply:
Just to be more organized and clear about what the methods does

Reply:
IMO it is more organized when you export individual functions.. function names should be clear enough for explanation =) 

or should they be static methods of the `Sessions` class? idk.